docs/arch/adr-33.md

```diff
@@ -0,0 +1,29 @@
+# ADR 33: Survey Task Feedback
+
+July 21, 2021
+
+## Context
+
+We plan to implement feedback for the survey task.
+
+Currently in PFE, the only feedback related to the survey task is for the Gravity Spy project, and is referred to as "Gravity Spy Gold Standard". Gravity Spy Gold Standard is a PFE experimental feature (per workflow configuration object) that checks a subject's metadata for `'#Type' === 'Gold'` and provides feedback after classification completion with the [`gs-gold-standard-summary`](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/gs-gold-standard-summary.jsx). Survey task annotation values are compared to subject metadata `'#Label'` values. A message is then shown accordingly (i.e. success, failure, or special message per subject metadata `'#post_classification_feedback'`).
+
+## Decision
+
+To implement a survey task feedback strategy within [the existing feedback framework](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/src/store/feedback/strategies). The initial survey task annotation feedback will presume the annotation has one annotation value per survey task choice. The subject metadata will include:
+
+- #feedback_N_id (required) - ID of the corresponding workflow task rule.
+- #feedback_N_choice_labels (required) - comma separated target choice label(s) (not values, i.e. "Blip" not "BLP").
```

Comment:
@lcjohnso great question:
> 1. Is there a specific reason for having the `#feedback_N_choice_labels` be the user-provided label (e.g., "Blip", "None of the Above") instead of the processed choice (e.g., "BLIP", "NONEOFTHEABOVE")?  I could see this being confusing, as Caesar uses the processed choice names for defining rules.  Also, the possibility of special characters in the label and the assumption that the `en`-version of the label is the one that should be provided are two extra complications, too.

I didn't realize Caesar was using the processed choice ID. I don't have a strong opinion either way. 

I've initially noted user provided labels because I thought it would be straightforward from the labels noted in the CSVs, whereas the processed ID takes checking the Raw task data or other steps. 

Any inconsistency between configurations is certain to cause issues though.

If users are already comfortable using the choice processed ID then I lean towards that for consistency, but I'm curious what others think?

Reply:
Noting for reference here's the PFE survey task editor code that converts user provided labels to the processed choice ID - https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/tasks/survey/editor.cjsx#L382